src/components/Card.tsx

```diff
@@ -0,0 +1,18 @@
+import Link from 'next/link';
+
+import { ICard } from '../models/Card';
+
+export function Card(props: { card: ICard }) {
+    return (
+        <Link
+            href={`/cards/${props.card.id}`}
+            className="w-80 h-52 overflow-hidden rounded-3xl border-2 -mt-36 first:mt-0"
+        >
+            <img
+                srcSet="https://placehold.co/600x400?text=Placeholder" 
```

Comment:
[NIT] The src should be from the props. Implementing it like this may be better.
```js
const { id, image_front } = props.card;
...
srcSet={image_front || "https://placehold.co/600x400?text=Placeholder"} 
```